package.json

```diff
@@ -9,7 +9,7 @@
     "prepare": "husky install",
     "postinstall": "patch-package",
     "ci": "turbo run lint build test",
-    "start": "turbo run start --parallel",
+    "start": "turbo run start --parallel --filter=app --filter=backend",
```

Comment:
Knowing that starting the frontend separately from the backend will not manager dynamic plugin config scehams correctly, due to webpack being used (if it is still the case), we might even want to replace the start command by:

```
turbo run build --filter=app && turbo run start --filter=backend
```

That builds the frontend app and thus also generates the frontend app static assets.
Then the backend app, detecting the static assets, serves the frotnend application through the `app-backend` plugin.
This has the double advantage of:
- automatically serving the frontend app as part of the backend
- Correctly serve to the frontend app the app config files of all the dynamic plugins known by the backend.

Reply:
Ok confirmed that it works, However had to update the `app-config.yaml` a bit. I had to update the `app.baseUrl` to `http://localhost:7007` and the remove the `backend.cors.origin` line since I don't think it would be needed anymore.
cc @04kash